src/styles.css

```diff
@@ -1,3 +1,7 @@
+body {
+    overflow-x: hidden;
+}
```

Comment:
We cannot do this. This will affect the user's website. We can increase the buffer if needed.

Reply:
ok I will consider this as well. Will update code